### ISSUE TITLE
Feature Request: Support Hard Line Breaks in Markdown

### ISSUE THREAD

**Issue by tilman-schieber** [LiaScript/LiaScript#189]
## Summary
Add hard line breaks to the markdown parser.  

## Reasoning
* Some content needs non-paragraph linebreaks (e.g. Poetry)
* It is a common feature of different Markdown dialects ([Github flavored markdown](https://github.github.com/gfm/#hard-line-breaks), [CommonMark](https://spec.commonmark.org/0.17/#hard-line-breaks))  
therefore not supporting it breaks user expectation *(at least mine)* 

## Proposed Solution
CommonMark and Github's MD both use the following convention
1. two spaces followed by a newline is translated to `<br />`
2. a backslash followed by a newline is translated to `<br />`

**Comment by andre-dietrich** [comment]
I added this feature to the LiaScript parser .... I will still need to run a few tests, but I think we will release by the end of this week ;-)

**Comment by tilman-schieber** [comment]
Thanks for the quick implementation. Just for clarity's sake:

> 1. two spaces followed by a newline is translated to <br />
> 2. a backslash followed by a newline is translated to <br />

part 2 is now implemented by eaa6f3e
I think it is fine not to implement the *spaces to line breaks*-part. invisible markup is not the greatest idea.

**Comment by andre-dietrich** [comment]
With the latest update discovered, that now it is also possible to break headings. Do you know of other Markdowns, that support this? GitHub does not as I checked, but this would be a really nice feature to deal with long and short section-names as it is also supported by latex. The first is the title in the table-of-contents, and the second and further parts are displayed in the slide ... Just thinking aloud ... 


``` markdown
# Hello World \
 with a longer url

This is a paragraph\
with a line-break...

[next](#a-section-with-a-very-very-long-title)

## A Section\
 with a very very long title

```

[Demo](https://liascript.github.io/LiveEditor/?/show/code/H4sIAAAAAAAAA0XNvQoCMRAE4H6fYiCNFpt3sLNXsPAsVl0uwZAcezl/3l5jEGGY6mPGYaspFRyKpSsGwiPWAEEqeVTDYoloH+KMTwSTmIwmUxjo52JWPpvKzXtPdMz6rKeVE571UmPJ3BwL39Vevdoy11iTromcwwa7Tv/nzfVqGF9M9AaYGlNGqwAAAA==)
